src/guide/jbs-jdk-bug-system.md

```diff
@@ -431,7 +431,7 @@ This table contains some frequently used JBS labels and their meaning. Please he
                            [*(Rel)*[-fix-yes]{.jbs-label}]{#rel-fix-yes}<br />
                            [*(Rel)*[-fix-no]{.jbs-label}]{#rel-fix-no}</td>
     <td class="dictionary">
-      Used in ramp down phase 2 to indicate that an issue would be of interest to get integrated into release *(Rel)*. E.g., [jdk12u-fix-request]{.jbs-label}
+      Used in ramp down phase 2 to indicate that an issue would be of interest to be integrated into release *(Rel)*. E.g., [jdk12u-fix-request]{.jbs-label}
```

Comment:
The `jdk*u-fix-request` labels are used for approval to backport / integrate a fix into an update release rather than after ramp down phase 2. However, the labels without `*u` in release family apply to RDP2.

Also, below sentences mention the label explicitly, this one should do it too: `*(Rel)*[-fix-request]{.jbs-label}`. It'll make the reference explicit.

Reply:
I have made this fix as part of #146. Please have a look to make sure I got it right.

Reply:
Shall this `s/get/be/` dropped from this PR to avoid conflict in #146?